content/docs/tutorials/kubernetes/guestbook.md

```diff
@@ -619,7 +619,7 @@ func main() {
 
         // Frontend Service
         var frontendServiceType string
-        if isMinikube {
+        if useLoadBalancer {
```

Comment:
Flip branch - i.e. `LoadBalancer` when `useLoadBalancer` is true and `ClusterIP` otherwise.